src/content/en/updates/2018/06/feature-policy.md

```diff
@@ -0,0 +1,336 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Feature Policy allows developers to selectively enable, disable, and modify the behavior of certain APIs and features in the browser. It's like CSP, but for features! Shipped in Chrome 60.
+
+{# wf_updated_on: 2018-06-22 #}
+{# wf_published_on: 2018-06-22 #}
+{# wf_tags: ux,feature,chrome60,feature-policy #}
+{# wf_featured_image: /web/updates/images/generic/checklist.png #}
+{# wf_featured_snippet: Feature Policy allows developers to selectively enable, disable, and modify the behavior of certain APIs and features in the browser. It's like CSP, but for features! Shipped in Chrome 60. #}
+{# wf_blink_components: Blink>FeaturePolicy #}
+
+# Introduction to Feature Policy {: .page-title }
+
+{% include "web/_shared/contributors/ericbidelman.html" %}
+
+<style>
+figure {
+  text-align: center;
+}
+figcaption {
+  font-style: italic;
+}
+</style>
+
+#### TL;DR {: #tldr .hide-from-toc}
+
+[Feature Policy][spec] allows web developers to selectively enable, disable, and
+modify the behavior of certain APIs and web features in the browser. __It's like
+[CSP](/web/fundamentals/security/csp/) but instead of controlling security, it's
+controls features!__
+
+The feature policies themselves are little opt-in aggreements between developer
+and browser that can help foster our goals of building (and maintaining) high
```

Comment:
'our' in this sentence will be interpreted as Google. (I'm not sure if that's what you intended.) Since we want this to be a platform feature, I'd stay away from such language.

Reply:
What's a better way to communicate that this helps our (us web devs, including myself) collective goal of building better apps?

Reply:
Try this:

'that can help improve the quality of web apps for the whole platform'